app/controllers/spree/admin/resource_controller.rb

```diff
@@ -251,8 +251,11 @@ def object_url(object = nil, options = {})
         end
       end
 
+      # Permit specific list of params
+      #
+      # Example: params.require(object_name).permit(:name)
       def permitted_resource_params
-        params.require(object_name).permit!
+        raise "All extending controllers need to override the method permitted_resource_params"
```

Comment:
:muscle:

Reply:
I have 37 exceptions in the latest build... I'll fix them now.

Reply:
ok, all exceptions fixed now with a few more simple commits to review :-)